lib/galaxy/datatypes/converters/fastagz_to_fasta.xml

```diff
@@ -0,0 +1,11 @@
+<tool id="CONVERTER_fastqgz_to_fastq" name="Convert fastq.gz files to fastq" version="1.0.0" hidden="true">
+  <command>gzip -dcf '$input1' > '$output1'</command>
```

Comment:
would be nice to have a requirement/dependency here on gzip ...

Reply:
we can do that

Reply:
well ... you mean from conda?

Reply:
fastq.gz converters do not have that dependency either

Reply:
I'm not sure why we do not have this, but I think it would be good if we have a requirement defined :)

Reply:
Here's why: https://github.com/galaxyproject/galaxy/pull/3145#discussion_r87689662